cv_next/app/login/page.tsx

```diff
@@ -0,0 +1,19 @@
+"use client"
+import { Auth } from "@supabase/auth-ui-react"
+import { useSupabase } from "@/hooks/supabaseHooks"
+import { ThemeSupa } from "@supabase/auth-ui-shared"
+
+export default function Page() {
+  const supabase = useSupabase()
+
+  return (
+    <Auth
+      supabaseClient={supabase}
+      appearance={{ theme: ThemeSupa }}
+      theme="dark"
+      providers={["google"]}
+      redirectTo="http://localhost:3000/auth/callback"
```

Comment:
Don't merge with localhost urls

Reply:
If you want, the only place localhost urls are allowed is in a development env file. You're more than welcome to add it and add a run config with the dev env.

Reply:
So what should I put there for the time being? Should I put the URL as a process env?

Reply:
Exactky, urls should be constants and urls that depends on env should be process env constants